src/coreclr/pal/src/misc/sysinfo.cpp

```diff
@@ -484,25 +484,6 @@ GlobalMemoryStatusEx(
     return fRetVal;
 }
 
-PALIMPORT
-DWORD
-PALAPI
-GetCurrentProcessorNumber()
-{
-#if HAVE_SCHED_GETCPU
```

Comment:
Can this be deleted too?
```
C:\runtime\src\coreclr\nativeaot\Runtime\unix\config.h.in(20):#cmakedefine01 HAVE_SCHED_GETCPU
C:\runtime\src\coreclr\nativeaot\Runtime\unix\configure.cmake(65):}" HAVE_SCHED_GETCPU)
C:\runtime\src\coreclr\pal\src\config.h.in(111):#cmakedefine01 HAVE_SCHED_GETCPU
C:\runtime\src\coreclr\pal\src\configure.cmake(387):}" HAVE_SCHED_GETCPU)
```

Reply:
Right, only GC and Libraries need these now. Other instances can be removed.